source/content/guides/wordpress-composer/04-bedrock-composer.md

```diff
@@ -0,0 +1,75 @@
+subtitle: Create a Composer-managed WordPress Site with Bedrock
+description: Learn more about Bedrock for Composer-managed WordPress sites.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/bedrock-composer
+anchorid: bedrock-composer
+---
+
+[Composer](https://getcomposer.org/) does not fully support WordPress, however, Bedrock is a central feature of Composer. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
+
+## Create a New Pantheon WordPress Site with Bedrock
+
+1. Run the following command in Terminus:
+
+    ```bash
+    composer create-project your_site
+    ```
+
+1. Update the [environment variables](https://docs.roots.io/bedrock/master/environment-variables/#wp-env) in your project's `.env` file.
+
+1. Store your `.env` outside of the project root on the Pantheon server. 
+
+    - Bedrock is configured to look for the `.env` file in the `files/private` directory by default. This is also the location where Pantheon stores all files that shouldn't be accessible to the public.
+
+1. Commit the `vendor` and `web` directories.
+
+    - Pantheon does not support using Composer to install dependencies that are deployed to Pantheon servers. You must commit the entire `vendor` and `web` directories into your Git repo to ensure that your dependencies are installed for the new project.
+
+1. Convert the `web/app/uploads` directory to a [symlink](https://github.com/git-for-windows/git/wiki/Symbolic-Links) that points to the `files` directory. 
+
+    - This is necessary because Pantheon stores media files in the `files` directory instead of the `uploads` directory.
+
+1. Create the initial `.env` file.
+
+    - This is necessary because Bedrock uses environment variables to manage sensitive credentials. These environment variables usually come from multiple sources. The most important variable is the `.env` file which Bedrock uses instead of the standard WordPress `wp-config.php` file. Pantheon was not designed to create the initial `.env` file that your Bedrock site requires. You must create it yourself and upload it to the Pantheon server. The easiest way to do that is by connecting to your Pantheon server using FTP and creating the `.env` file. 
+
+1. Upload the newly created `.env` file by connecting to the Pantheon server using an FTP.
+
+    - To find your connection information: Navigate to the site's **Dev** environment in Pantheon > click **Connection Info** > scroll to the bottom of the menu to view your SFTP connection information.
+
+1. Connect to your Pantheon site using the SFTP credentials located in the previous step > navigate to the `files` directory > create a `private` directory > create the `.env` file in the `private` directory by running the following command:
+
+    ```bash
```

Comment:
It might be more helpful to provide an example of what the `.env` file looks like since this looks like Bedrock default. None of the credentials are exposed because we use global server variables.

```bash
DB_NAME=$_ENV['DB_NAME']
DB_USER=$_ENV['DB_USER']
DB_PASSWORD=$_ENV['DB_PASSWORD']

# Optionally, you can use a data source name (DSN)
# When using a DSN, you can remove the DB_NAME, DB_USER, DB_PASSWORD, and DB_HOST variables
# DATABASE_URL='mysql://database_user:database_password@database_host:database_port/database_name'

# Optional database variables
DB_HOST=$_ENV['DB_HOST'].':'.$_ENV['DB_PORT']
# DB_PREFIX='wp_'

WP_ENV='development'
WP_HOME='http://example.com'
WP_SITEURL="${WP_HOME}/wp"

# Specify optional debug.log path
# WP_DEBUG_LOG='/path/to/debug.log'

# Generate your keys here: https://roots.io/salts.html
AUTH_KEY=$_ENV['AUTH_KEY']
SECURE_AUTH_KEY=$_ENV['SECURE_AUTH_KEY']
LOGGED_IN_KEY=$_ENV['LOGGED_IN_KEY']
NONCE_KEY=$_ENV['NONCE_KEY']
AUTH_SALT=$_ENV['AUTH_SALT']
SECURE_AUTH_SALT=$_ENV['SECURE_AUTH_SALT']
LOGGED_IN_SALT=$_ENV['LOGGED_IN_SALT']
NONCE_SALT='generateme'
```